File: lib-base/src/main/java/com/llj/lib/base/IFragmentLazy.java
package com.llj.lib.base;

/**
 * ArchitectureDemo
 * describe:
 * author liulj
 * date 2018/5/24
 */
public interface IFragmentLazy {

    default boolean useLazyLoad() {
        return false;
    }

    boolean hasInitAndVisible();

    default void onLazyLoad() {

    }

}

File: lib-base/src/main/java/com/llj/lib/base/IFragmentLazy.kt
package com.llj.lib.base

/**
 * ArchitectureDemo
 * describe:
 * author liulj
 * date 2018/5/24
 */
interface IFragmentLazy {

    fun useLazyLoad(): Boolean {
        return false
    }

    fun hasInitAndVisible(): Boolean

    fun onLazyLoad() {

    }
}
